docs: extended Prometheus guide to cover Kiali (#2668)
kyma-project/telemetry-manager: docs/user/integration/prometheus/README.md

The Telemetry module supports shipping metrics from applications and the Istio service mesh to Prometheus using the OpenTelemetry protocol (OTLP). Prometheus is a widely used backend for collection and storage of metrics. To provide an instant and comprehensive monitoring experience, the `kube-prometheus-stack` Helm chart bundles Prometheus together with Grafana and the Alertmanager. Furthermore, it brings community-driven best practices on Kubernetes monitoring, including the components `node-exporter` and `kube-state-metrics`.

> Because the OpenTelemetry community is not that advanced yet in providing a full-blown Kubernetes monitoring solution (without `node-exporter` as additional tools), this guide shows how to combine the two worlds by integrating application and Istio metrics based on the Telemetry module, and the Kubernetes monitoring based on the features of the bundle.

Reviewer: ```suggestion
Because the OpenTelemetry community doesn't provide a full-blown Kubernetes monitoring solution yet (without `node-exporter` as additional tools), this guide shows how to combine the two worlds by integrating application and Istio metrics based on the Telemetry module, and the Kubernetes monitoring based on the features of the bundle.
```
